apps/api/src/deployment/controllers/deployment/deployment.controller.ts

```diff
@@ -1,15 +1,24 @@
+import { SDL } from "@akashnetwork/akashjs/build/sdl";
+import assert from "http-assert";
 import { singleton } from "tsyringe";
 
+import { AuthService, Protected } from "@src/auth/services/auth.service";
+import { UserWalletRepository } from "@src/billing/repositories";
+import { CreateDeploymentRequest, CreateDeploymentResponse, SingleDeploymentResponse } from "@src/deployment/http-schemas/deployment.schema";
+import { DeploymentService } from "@src/deployment/services/deployment/deployment.service";
 import { StaleManagedDeploymentsCleanerService } from "@src/deployment/services/stale-managed-deployments-cleaner/stale-managed-deployments-cleaner.service";
 import { TopUpManagedDeploymentsService } from "@src/deployment/services/top-up-managed-deployments/top-up-managed-deployments.service";
 import { TopUpDeploymentsOptions } from "@src/deployment/types/deployments-refiller";
 import { CleanUpStaleDeploymentsParams } from "@src/deployment/types/state-deployments";
 
 @singleton()
-export class TopUpDeploymentsController {
+export class DeploymentController {
```

Comment:
non-blocking: why do you think we should merge these? Why not having 2 specific controllers?